src/gui/InventoryWindowController.java

```diff
@@ -51,8 +51,8 @@ private void onUseClicked(ActionEvent event) {
      */
     @FXML
     private void onDropClicked(ActionEvent event) {
-        game.command("drop", listView.getSelectionModel().getSelectedItem().replace(" ", "-"));
-        addItemsToViewList();
+            game.command("drop", listViewSelection(listView));
+            addItemsToViewList();
```

Comment:
Indentation

Reply:
fixed-in: 00432b3a4c70b92bf2e0734f218be5e4bb834768